ledger/trie/trie.go

```diff
@@ -176,8 +177,10 @@ func (t *Trie) Mutate(paths []ledger.Path, payloads []ledger.Payload) (*Trie, er
 				}
 			}
 
-			// At this point, we are definitely going to create a branch.
+			// At this point, we are definitely going to create a branch. This
+			// means the extension we are modifying is no longer ceal either.
```

Comment:
// means the extension we are modifying is no longer ceal either. -> "clean"

Reply:
Outdated as well